### ISSUE TITLE
Unable to configure custom domain past MX record

### ISSUE THREAD

**Issue by fourtwizzy** [CTemplar/webclient#1657]
Your environment

macOS 12.1
Firefox v97.0
Chromium V100.0.4888.0 
Safari v15.2 (17612.3.6.1.6)

Steps to reproduce

1. Add custom domain, add TXT record like asked. Comes back as domain verified.
2. Add MX record. This continually fails even though it shows as done.
3. Eventually you will just get blank results.

TL;DR: Setup TXT, MX records, unable to proceed to SPF and beyond.

Expected behavior

Setting up a custom domain without issue

### Actual behavior

No matter what browser I use it the website even though claiming my MX record is done (which it is), will not allow me to progress further. I did see a  #comparable issue to this, and wondered if it was related. https://github.com/CTemplar/webclient/issues/1319 

 #1319

<img width="1152" alt="Screen Shot 2022-02-13 at 4 53 21 PM" src="https://user-images.githubusercontent.com/1735941/153777777-94641b15-de9e-42f0-9cc7-8d55746db697.png">
<img width="1149" alt="Screen Shot 2022-02-13 at 4 53 36 PM" src="https://user-images.githubusercontent.com/1735941/153777781-39793d70-5c8f-4c3c-9451-d499903f0de1.png">
<img width="1132" alt="Screen Shot 2022-02-13 at 4 53 46 PM" src="https://user-images.githubusercontent.com/1735941/153777782-453dca19-73d8-4ed6-ae28-72ef3dfc317b.png">
<img width="1202" alt="Screen Shot 2022-02-13 at 10 09 11 AM" src="https://user-images.githubusercontent.com/1735941/153777783-072f2636-3522-41a0-9e77-1b0268f62954.png">

**Comment by The-Hidden-Hand** [comment]
Thanks for the report and our apologies, we are investigating and gonna solve it soon.

Duplicates: https://github.com/CTemplar/support/issues/469

**Comment by The-Hidden-Hand** [comment]
I've tested with a domain of my own, and I can navigate all the screens and see al the information of the domain again.

I had to click the edit button:
![image](https://user-images.githubusercontent.com/45569116/153854403-28e6c58e-423e-450f-9173-f42e80677640.png)

**Comment by Ogheneruona2** [comment]
I saw the comments

**Comment by The-Hidden-Hand** [comment]
Good job guys

**Comment by The-Hidden-Hand** [comment]
@fourtwizzy the patch has been released. Please let us know if it's still not working or you have any other concerns.

**Comment by fourtwizzy** [comment]
@The-Hidden-Hand  Issue still isn't resolved on ctemplar.com.  I still could not progress past the MX record. As such I deleted the domain and decided to start over again. Now I've made it to the SPF record is good, but the MX record still fails verification. Looking at the response code, it still tells me I have exceeded my domain limit. 

P.S if you are wondering how I managed to put in the SPF record without getting past the MX record, I figured last night I could see the appropriate DKIM, DMARC, SPF values in the JSON so I added them to my DNS records hoping that a quick reverify would notice everything is there. Alas, I still somehow have more than one domain.

![Screenshot 2022-02-17 085952](https://user-images.githubusercontent.com/1735941/154496959-28054968-cde0-486f-ae20-3781a881870d.png)
.
